spring-cloud-kubernetes-fabric8-discovery/src/main/java/org/springframework/cloud/kubernetes/fabric8/discovery/Fabric8KubernetesDiscoveryClientUtils.java

```diff
@@ -64,11 +64,11 @@
 /**
  * @author wind57
  */
-final class KubernetesDiscoveryClientUtils {
```

Comment:
I renamed this one to `Fabric8KubernetesDiscoveryClientUtils`, since it's not public, that is OK. This triggered some other changes of course